op-bindings/predeploys/addresses.go

```diff
@@ -25,8 +25,8 @@ const (
 	L1FeeVault                    = "0x420000000000000000000000000000000000001a"
 	SchemaRegistry                = "0x4200000000000000000000000000000000000020"
 	EAS                           = "0x4200000000000000000000000000000000000021"
-	CeloRegistry                  = "0x4200000000000000000000000000000000000022"
-	GoldToken                     = "0x4200000000000000000000000000000000000023"
+	CeloRegistry                  = "0x000000000000000000000000000000000000ce10"
+	GoldToken                     = "0x471ece3750da237f93b8e339c536989b8978a438"
```

Comment:
This is the current mainnet address. I like this choice, since we will maintain the current address on mainnet and get a fixed address across all cel2 networks.